packages/malloy-db-test/src/nomodel.spec.ts

```diff
@@ -150,6 +175,31 @@ runtimes.runtimeMap.forEach((runtime, databaseName) => {
     expect(result.data.value[0].c).toBe(19701);
   });
 
+  it(`join_one filter multiple values - ${databaseName}`, async () => {
+    const result = await runtime
+      .loadQuery(
+        `
+      explore: a is table('malloytest.state_facts'){
+        where: state = 'TX' | 'LA'
+      }
+      explore: b is table('malloytest.airports') {
+        join_one: a on state=a.state
+      }
+      query: b->{
+        aggregate: c is a.airport_count.sum()
+        group_by: a.state
+      }
+      `
+      )
+      .run();
+    expect(result.data.value[0].c).toBe(1845);
+    expect(result.data.value[0].state).toBe("TX");
+    expect(result.data.value[1].c).toBe(500);
+    expect(result.data.value[1].state).toBe("LA");
+    expect(result.data.value[2].c).toBeNull();
+    expect(result.data.value[2].state).toBeNull();
```

Comment:
Hmmm ... Postgres sorts it the opposite way. Will look into why.

Reply:
Ah, when it's PG, when need to do `NULLS LAST` when ordering descending. It also works in BigQuery.

Reply:
Put a test for now that doesn't consider the order, since BQ and PG sort nulls differently. Depending on whether Malloy should always sort nulls the same way regardless of language, this could be split into its own issue and these tests changed back to check the order after that.

https://cloud.google.com/bigquery/docs/reference/standard-sql/query-syntax#order_by_clause
> If null ordering is not specified with NULLS FIRST or NULLS LAST:
> NULLS FIRST is applied by default if the sort order is ascending.
> NULLS LAST is applied by default if the sort order is descending.

https://www.postgresql.org/docs/current/queries-order.html
> By default, null values sort as if larger than any non-null value; that is, NULLS FIRST is the default for DESC order, and NULLS LAST otherwise.